ProblemsSolution/ProblemsSolution.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace ProblemsSolution
+{
+    public class ArrayProblemsSolution
+    {
+        public static int SumArrayEvenNumbers(int[] numbers)
+        {
+            int sum = 0;
+            foreach(int number in numbers)

```

Comment:
This might be simplified. Ever heard of [map](https://blog.submain.com/csharp-functional-programming/)?

Reply:
I wasn't sure if I was allowed to use LINQ or not so I decided to do it the tedious way. Also not sure how would I do this with map? I'm thinking about filtering even numbers and then reducing with a sum function.

Reply:
> This might be simplified. Ever heard of [map](https://blog.submain.com/csharp-functional-programming/)?

I've simplified it here aefbbf8c1c711595bcbe35054bb1b342993a8d8c